auth/active_directory.adoc

```diff
@@ -77,6 +77,8 @@ Password for user: xxxxxxxx
 
 Update the */etc/sssd/sssd.conf* file as follows:
 
+*Note*: Starting with SSSD version 1.15.2, which will be available in CentOS version 7.4, SSSD will provided the domain name as a user attribute. The below examples show how to set _ldap_user_extra_attrs_ and _user_attributes_ to take advantage of this new feature. If running an appliance built with CentOS version prior to CentOS 7.4 do not include _domainname_ for these attributes.
```

Comment:
Typo - `will provided` should be `will provide`

Reply:
@gtanzillo Thank you. Good catch!